### PR TITLE
Change to the cooler design!

### PR DESCRIPTION
**Pull request by Shinaco1992**
Related to #1.

- Don't use bootstrap.
- Image sizes and messages are displayed at the bottom of the preview area in one place.

It's a rough change.
The following issues still remain.

- Input form design
- Design of grayscale and reset
- Expansion of drop area

Please review once.

### CODE REVIEW

File: docs/drop-zone.js [hunk 1]
```diff
@@ -9,30 +9,28 @@ window.onload = function () {
...
-        imageStatus.innerHTML = "<h2>upload image</h2>"
+        imageStatus.innerHTML = "upload image"
         errorMessage.innerHTML = ""
     })
 
     dropZone.addEventListener('drop', function (e) {
         e.stopPropagation()
         e.preventDefault()
-        this.style.background = '#ffffff'
+        // this.style.background = '#ffffff'
```
Comment (icemint0828) [inline docs/drop-zone.js]: It seems unnecessary, so I'll delete it.

File: docs/index.html [hunk 2]
```diff
@@ -2,84 +2,67 @@
 <head>
     <meta charset="UTF-8">
     <title>imgedit</title>
-    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
+<!--    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">-->
```
Comment (icemint0828) [inline docs/index.html]: It seems unnecessary, so I'll delete it.

File: docs/index.html [hunk 3]
```diff
@@ -2,84 +2,67 @@
...
     <link rel="stylesheet" type="text/css" href="wasm.css">
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
+<!--    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>-->
```
Comment (icemint0828) [inline docs/index.html]: It seems unnecessary, so I'll delete it.

**Review by icemint0828 — COMMENTED** [review]
In fact, the merge is performed because the page cannot be verified without merging.

File: docs/drop-zone.js (hunk 1 above)
Comment (Shinaco1992) [reply docs/drop-zone.js]: Thank you very much!

File: docs/index.html (hunk 2 above)
Comment (Shinaco1992) [reply docs/index.html]: Thank you very much!

File: docs/index.html (hunk 3 above)
Comment (Shinaco1992) [reply docs/index.html]: Thank you very much!

**Comment by icemint0828** [comment]
@Shinaco1992 
You did such a great job on this project!

Much of the evaluation of this project relies on index.html, not main.wasm.
I am not very skilled in handling CSS and javascript, so it was difficult for me to express this.

Please continue with the rest of the work!

**Pull request merged**